sovtokenfees/sovtokenfees/test/catchup/revert/test_revert_fees.py

```diff
@@ -0,0 +1,84 @@
+from threading import Thread
+
+import pytest
+from sovtoken.constants import XFER_PUBLIC, ADDRESS, AMOUNT, SEQNO, OUTPUTS
+from sovtokenfees.constants import FEES
+from sovtokenfees.test.helper import check_state, ensure_all_nodes_have_same_data, send_and_check_nym_with_fees
+
+from indy_common.constants import NYM
+from plenum.common.startable import Mode
+from plenum.common.txn_util import get_seq_no
+from plenum.test.delayers import cDelay
+from plenum.test import waits
+from plenum.test.helper import assertExp
+from plenum.test.stasher import delay_rules
+from plenum.test.test_node import ensureElectionsDone
+from plenum.test.view_change.helper import ensure_view_change
+from stp_core.loop.eventually import eventually
+
+
+def test_revert_fees_with_xfer(nodeSetWithIntegratedTokenPlugin, xfer_mint_tokens,
```

Comment:
I think a better name for the test can be `test_revert_during_view_change_all_nodes_xfer_with_fees`